Automattic/wp-calypso
# Update Jetpack cloud to be more in line with calypso

This PR removes cloud specific layout controllers and replaced them with calypso specific ones.

In https://github.com/Automattic/wp-calypso/pull/40022 we started using the calypso specific entry point. 

This PR continuous this work by removing the jetpack cloud specific functions that are replicas of what is already available in calypso. 

Before:
<img width="457" alt="Screen Shot 2020-03-12 at 2 21 57 PM" src="https://user-images.githubusercontent.com/115071/76612488-b4800580-651c-11ea-818b-fa0a83c87c38.png">

After:
<img width="976" alt="Screen Shot 2020-03-12 at 2 39 48 PM" src="https://user-images.githubusercontent.com/115071/76612500-bd70d700-651c-11ea-9b13-ba211a302126.png">


#### Changes proposed in this Pull Request
* Removes the Jetpack Cloud specific entrypoint functions.

#### Testing instructions
* Load up calypso `npm start`. Does it still look the same as before? Any regressions? 
* Load up jetpack cloud with `npm run start-jetpack-cloud` 
- Notice that the masterbar is always what you expect during the loading process. 
- Notice the sidebar has the sites selector. 
- Notice that when you visit a section without the site selected you end up seeing the site selector.
